incidents/2020-03-24--iiif-prod-outage.md

```diff
@@ -0,0 +1,49 @@
+# iiif outage
+
+* affected iiif--prod--* 
+* affected for approximately 4 hours.
```

Comment:
can be separated into `Impact`, see https://github.com/elifesciences/tech-team/blob/master/incidents/2018-04-24-journal-outage.md#impact